BabbleApp/camera.py

```diff
@@ -216,10 +219,9 @@ def get_serial_camera_picture(self, should_push):
                 jpeg = self.get_next_jpeg_frame()
                 if jpeg:
                     # Create jpeg frame from byte string
-                    image = cv2.imdecode(
-                        np.fromstring(jpeg, dtype=np.uint8), cv2.IMREAD_UNCHANGED
-                    )
-                    if image is None:
+                    try:
+                        image = np.array(Image.open(BytesIO(jpeg)))
```

Comment:
Seems to work as expected although is there a reason why imdecode was replaced with a method from pillow?